orion/sites/admin/src/announce/announce.js

```diff
@@ -3,11 +3,66 @@ require('./announce.styl');
 import React from 'react';
 import ReactDOM from 'react-dom';
 
+
 export class AnnouncePage extends React.Component {
 	render() {
+		var numAnnouncements = 5; 	{/* this needs to be a variable later */}
+
+		{/* these fake announcements will eventually be replaced with real announcements */}
+		var fakeAnnounceA = {
+			date: "02/15/2020",
+			author: "Melon King",
+			message: "Only evil people waste watermelon"
+		};
+		var fakeAnnounceB = {
+			date: "02/16/2020",
+			author: "Fruit Hater",
+			message: "Watermelons are fruit, therefore they deserve to dry on the vine"
+		};
+		{/* will use this to test text wrapping later */}
+		var fakeAnnounceC = {
+			date: "02/23/2020",
+			author: "Talkative",
+			message: "This will be a longer message later"
+		};
 		return (
       <div id="view-announce">
-        <h1>All Announcements</h1>
+        <h1>All Announcements ({numAnnouncements}) </h1>
+		<ul> {/* this is the subheading */}
```

Comment:
however these comments here are correct!

Reply:
In addition, for this subheading area, You have a `<ul>` with only one `<li>` in the list. I think instead of the `<ul>` and `<li>`, it might be simpler to have one `<div>` to encase the Date, Author and Message.